gdb-multi-arch/conda_build_config.yaml

```diff
@@ -1,5 +1,5 @@
 CONDA_BUILD_SYSROOT:
-  - /opt/MacOSX10.9.sdk        # [osx]
+  - /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk
```

Comment:
Undoing this. We stick to the conda-forge strategy of placing these SDK's at /opt/...

https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html#macos-sdk
https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L95-L97